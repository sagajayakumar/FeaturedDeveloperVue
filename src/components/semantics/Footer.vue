<template>
  <div class="footer">
  <div><a href="https://github.com/KeerthiMuli/Featured-Developer-Proposal"> 
  <font-awesome-icon :icon="['fab', 'github']" class="icon alt"/> </a> &nbsp;&nbsp; 

  <a href="https://github.com/sagajayakumar/FeaturedDeveloperVue"> 
  Front End </a> &nbsp;&nbsp;

  <a href="https://github.com/KeerthiMuli/Featured-Developer-ThreeB-backend"> 
  Back End </a> &nbsp;&nbsp;

  <a href="https://github.com/KeerthiMuli/Featured-Developer-Proposal/projects/1"> 
  Board </a> &nbsp;&nbsp;

  <a href="https://featured-developers-threeb.herokuapp.com/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config"> 
  Swagger </a> &nbsp;&nbsp;
  
  <a href="https://github.com/KeerthiMuli/Featured-Developer-Proposal/wiki"> 
  Personal Wikis </a> &nbsp;&nbsp;

   <a href="https://sagajayakumar.github.io/featured_developer_code_coverage/"> 
  Code Coverage </a> &nbsp;&nbsp;
  
   Featured Developers</div>
    </div>
  
</template>
<script>
import { defineComponent } from "@vue/composition-api";

export default defineComponent({
  plugins: ["~/plugins/font-awesome"],
});
</script>


<style scoped>
.footer {
  background-color: #006747;
  text-align: right;
  color: whitesmoke;
  padding: 10px;
  margin-top: 110px;
}
a {
  color: white;
  background-color: transparent;
  text-decoration: none;
}
a:hover {
  color: #5d63b8;
}
</style>