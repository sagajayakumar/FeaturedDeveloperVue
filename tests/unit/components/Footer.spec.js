import { mount } from '@vue/test-utils'
import Footer  from '../../../src/components/semantics/Footer.vue'


describe('Footer.vue', () => {
    test('snapshot', () => {
        const wrapper = mount(Footer)
        expect(wrapper).toMatchSnapshot()
    })

})