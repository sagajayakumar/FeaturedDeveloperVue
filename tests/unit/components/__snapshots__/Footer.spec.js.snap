// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue snapshot 1`] = `
<div class="footer">
  <div><a href="https://github.com/shivaramkothapally/featured-developers-proposal">
      <font-awesome-icon icon="fab,github" class="icon alt"></font-awesome-icon>
    </a> &nbsp;&nbsp;

    <a href="https://github.com/sagajayakumar/FeaturedDeveloperVue">
      Front End </a> &nbsp;&nbsp;

    <a href="https://github.com/KeerthiMuli/Featured-Developer-ThreeB-backend">
      Back End </a> &nbsp;&nbsp;

    <a href="https://trello.com/b/yRWaDdG1/featureddeveloper">
      Board </a> &nbsp;&nbsp;

    3A Featured Developers
  </div>
</div>
`;
